Dockerfile

```diff
@@ -87,51 +109,39 @@ WORKDIR     /root
 #           must mount ${SSH_AUTH_SOCK} to /ssh-agent to use host ssh
 RUN         mkdir -p /root/mounted
 ENV         VIRTUAL_ENV=/opt/venv \
-            SSH_AUTH_SOCK=/ssh-agent
+            SSH_AUTH_SOCK=/ssh-agent \
+            PYTHONDONTWRITEBYTECODE=1 \
+            PYTHONUNBUFFERED=1
+
 ENV         PATH="$VIRTUAL_ENV/bin:$PATH"
 COPY        --from=ansible $VIRTUAL_ENV $VIRTUAL_ENV
 RUN         apk --no-cache add \
               openssh-client~=9.1
-COPY        .flake8 ./.flake8
-COPY        .git/ ./.git
-COPY        .darglint ./.darglint
-COPY        canu/ ./canu
-COPY        docs/ ./docs
-COPY        network_modeling/ ./network_modeling
-COPY        pyinstaller_hooks/ ./pyinstaller_hooks
-COPY        tests/ ./tests
-COPY        canu.spec ./canu.spec
-COPY        canuctl ./canuctl
-COPY        Dockerfile ./Dockerfile
-COPY        entry_points.ini ./entry_points.ini
-COPY        LICENSE ./LICENSE
-COPY        Makefile ./Makefile
-COPY        MANIFEST.in ./MANIFEST.in
-COPY        mkdocs.yml ./mkdocs.yml
-COPY        noxfile.py ./noxfile.py
-COPY        pyinstaller.py ./pyinstaller.py
-COPY        pyproject.toml ./pyproject.toml
-RUN         PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 python -m pip install --no-cache-dir --editable '.[ci,docs]'
+RUN         python -m pip install --no-cache-dir --editable '.[ci,docs]'
 # hadolint ignore=DL3059
-RUN         nox -e docs
+RUN         sphinx-build -M markdown docs/templates docs/_build -a
```

Comment:
Yeah, this is way better and faster--I was going to do this direct instead of using `nox` but since so much was already changing with the advent of the container, I just left it in place.  I think if you are doing this, you can omit the `ci` extra from line 120...